### DISCUSSION TITLE
BR and TREL and other info

### DISCUSSION THREAD

**Discussion by agalliazzo** [openthread/openthread#9828] [General]
Hi Guys,
I recently added a second BR on my network and compile both my BR with the TREL support. It seems that it works, or at least, trel peers show each other BR and trel counters give numbers.

I have still a couple of questions:
1. Is it normal that one of the BR has a child role? does it works as BR also if not a router?
2. If BR is not in router role, is TREL still forwarding packet to the network properly?
3. Is there a way to force a BR to became a router?
4. If a leader of a partition is not able to see another partition, does the partitions finally merge?

Thanks in advance
Alessio

**Comment by abtink** [comment]
> Is it normal that one of the BR has a child role? does it works as BR also if not a router?

Yes. Device can act as Border Router which attached as leader, router, or child role within Thread mesh.

> If BR is not in router role, is TREL still forwarding packet to the network properly?

Yes. TREL is link-layer feature (multi-radio support), and the role of device does not matter.

> Is there a way to force a BR to became a router?

Role of devices within mesh are decided by the Thread network itself (there are rules for upgrade/downgrade, etc). 
Generally BRs wil request and become router after some time (this depends on number of other routes, etc).

> If a leader of a partition is not able to see another partition, does the partitions finally merge?

If two nodes from different partitions can communicate with each other, then partitions should merge. If partitions are fully isolated, they remain so.

**Reply by agalliazzo** [reply]
First of all, thanks for your prompt response!

Second, it's clear that I'm not able to count up to 4 cause I started with "a couple of questions" and the questions were 4...

Last, to better clarify my point, cause reading your answer is quite obvious that it should works as you say but I absolutley understand that my last question was too much generic.

What I intended to ask was: If I have a 2 different BR, with TREL enabled, then BR1 has it's own network on PanID=1234 and a PartitionId (the role is not important as you explained), and BR2 has it's own network too with the same dataset except for PartitionId. Will the 2 different partitions eventually joins also if no radio signal is exchanged between nodes? In other word, does a merge over TREL will be possible?

Thanks again
Alessio

**Reply by jwhui** [reply]
Yes, if the two BRs have the same Active Operational Dataset and are attached to the same infrastructure link, they should eventually combine into the same Thread Partition with the same Partition ID.

**Reply by agalliazzo** [reply]
Hi, thanks for the clarification. Really appreciated!
